axelar-chains-config/info/stagenet.json

```diff
@@ -1691,6 +1691,78 @@
+          ],
+          "initialVerifierSetId": "c7483b242156a8149fe6c4efcf62184d9fbf6b1be5530872f80a472d33070cf6",
+          "address": "0x5A4fA5187BddD93097dE4F6062a2E96C82Ea2d61",
+          "implementation": "0xC5578b1c63CE803B9f7f185652CC1E14Af887b8f",
+          "implementationCodehash": "0x3bd2a46c04ccf59a0e454ef828232d66a71c5a9b8004e0ea516257a7ad0aa21c",
+          "deploymentMethod": "create",
+          "previousSignersRetention": 15,
+          "domainSeparator": "0x3dad2b901117c30d4f601c3b9e9ee89cec4371e79e9908122e8e1da1904347ee",
+          "minimumRotationDelay": 300
+        },
+        "InterchainGovernance": {
+          "address": "0xBeF25f4733b9d451072416360609e5A4c115293E"
+        },
+        "ConstAddressDeployer": {
+          "address": "0x98B2920D53612483F91F12Ed7754E51b4A77919e",
+          "deployer": "0xE86375704CDb8491a5Ed82D90DceCE02Ee0ac25F",
+          "deploymentMethod": "create",
+          "codehash": "0x8fda47a596dfba923270da84e0c32a2d0312f1c03389f83e16f2b5a35ed37fbe",
+          "predeployCodehash": "0x8fda47a596dfba923270da84e0c32a2d0312f1c03389f83e16f2b5a35ed37fbe"
+        },
+        "Create3Deployer": {
+          "address": "0x6513Aedb4D1593BA12e50644401D976aebDc90d8",
+          "deployer": "0x6f24A47Fc8AE5441Eb47EFfC3665e70e69Ac3F05",
+          "deploymentMethod": "create2",
+          "codehash": "0xf0ad66defbe082df243d4d274e626f557f97579c5c9e19f33d8093d6160808b7",
+          "predeployCodehash": "0x73fc31262c4bad113c79439fd231281201c7c7d45b50328bd86bccf37684bf92",
+          "salt": "Create3Deployer"
+        },
+        "AxelarGasService": {
+          "collector": "0xc5C525B7Bb2a7Ce95C13Ee5aBdB7F8fd3cb77392",
+          "salt": "AxelarGasService",
+          "address": "0xeE1d04E4F9F9AdEB7be7d6996f4A726773460A87",
+          "implementation": "0x1F002cd258e2e818eC5ecCD46DD6cb938c5Cf7f8",
+          "deployer": "0xBeF25f4733b9d451072416360609e5A4c115293E"
+        },
+        "Operators": {
```

Comment:
has axelarons whitelisted their relayers on this contract? do we have a shared list of relayer addresses by network that we should be adding as operators during deployment? Will make the setup easier for axelarons

Reply:
We've been using our own gas service and operators contract for stagenet. If we should use the ones that are already deployed, we should update our configs.

To get the addresses of refunders per environment there's an axelarons command. Can also be found [here](https://github.com/axelarnetwork/axelarons-tools/blob/01076262318d60b88bba9a940f161a6ee20d7600/test-gmp/config/stagenet.json#L13) (it's the same set of addresses for all chains)

Reply:
Cool that's even better. We can refresh our configs so that we use the deployed operators contracts. On a second look, I think I've diverged from doing that with Flow - all the other chains use the same operators contract, which means it's the one deployed by chainops. Surely that's the case for testnet/mainnet, we don't deploy anything of our own there.

And yes, operators addresses are always the same across all chains.